spec/io/mat5_reader_spec.rb

```diff
@@ -0,0 +1,103 @@
+# = NMatrix
+#
+# A linear algebra library for scientific computation in Ruby.
+# NMatrix is part of SciRuby.
+#
+# NMatrix was originally inspired by and derived from NArray, by
+# Masahiro Tanaka: http://narray.rubyforge.org
+#
+# == Copyright Information
+#
+# SciRuby is Copyright (c) 2010 - 2014, Ruby Science Foundation
+# NMatrix is Copyright (c) 2012 - 2014, John Woods and the Ruby Science Foundation
+#
+# Please see LICENSE.txt for additional copyright notices.
+#
+# == Contributing
+#
+# By contributing source code to SciRuby, you agree to be bound by
+# our Contributor Agreement:
+#
+# * https://github.com/SciRuby/sciruby/wiki/Contributor-Agreement
+#
+# == fortran_format_spec.rb
+#
+# Basic tests for NMatrix::IO::Matlab
+#
+
+require_relative '../../lib/nmatrix'
+
+describe NMatrix::IO::Matlab do
+  def check_matrix_data_sparse arr
```

Comment:
I think you could use `shared_examples` instead of define a method. what do you think? 🤔